threadpoolctl.py

```diff
@@ -16,16 +16,11 @@
 import ctypes
 import warnings
 from ctypes.util import find_library
+from abc import ABC, abstractmethod
 
-__version__ = '1.2.0.dev0'
+__version__ = "1.2.0.dev0"
```

Comment:
Because this is a big refactoring we might want to target `__version__ = "2.0.0.dev0"` once this PR is merged, even if the backward compatibility should be preserved.